rafs/src/fs.rs

```diff
@@ -658,7 +658,7 @@ impl FileSystem for Rafs {
             let r = self.device.read_to(w, desc)?;
             result += r;
             recorder.mark_success(r);
-            if r as u32 != desc.size() {
+            if r as u64 != desc.size() {
```

Comment:
desc.size() is always `u32`. So this means that somehow we end up having `self.device.read_to` to return more than 4G. Kernel never sends such large read requests. So it means that we have expanded the read IO size to larger than 4G in the above `amplify_io` logic, which is definitely wrong.... Are you testing with an `amplify_io` config larger than 4G?

Also, I wonder if this gets fixed by #1311 as well.